vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/card/validation/rules/VGSInfoValidationRule.kt

```diff
@@ -0,0 +1,70 @@
+package com.verygoodsecurity.vgscollect.view.card.validation.rules
+
+class VGSInfoValidationRule(
+    regex: String?,
+    length:Array<Int>?
+): ValidationRule(regex, length)  {
+
+    /**
+     * This class provides an API for set up rules for validation person name.
+     */
+    open class ValidationBuilder {
```

Comment:
@Endorf I would suggest using `Builder` name, as `ValidationBuilder` is redundant as this class should be accessible line `VGSInfoValidationRule.Builder()`

Does `ValidationBuilder` should be an `open class`?

Reply:
ValidationBuilder naming context: I agree with you. But such naming is already presented in other classes which existed before. So, I decided to make naming consistent.